tools/istio-docker.mk

```diff
@@ -0,0 +1,209 @@
+## Copyright 2018 Istio Authors
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##     http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+
+.PHONY: docker docker.prebuilt
+
+docker: docker.all
+
+# Build docker images for pilot, mixer, ca using prebuilt binaries
+docker.prebuilt: docker.tag
```

Comment:
You can remove this target and simply update the .circleci/config.yaml file to point to the new targets..

Reply:
I was updating .circleci/config.yml to reflect your other PR (change "make go-build" to "make build") but there seems to be something odd about the line endings in the file--emacs thinks it's seeing DOS [i.e., Windows] line endings.  I don't think I want be listed as modifying the whole file so I'll just stick to the lines I'm touching.